libavfilter/dnn/dnn_backend_native_layer_mathbinary.h

```diff
@@ -47,6 +47,17 @@ typedef struct DnnLayerMathBinaryParams{
     float v;
 } DnnLayerMathBinaryParams;
 
+typedef float (*FunType)(const float src, float dst);
```

Comment:
it's not need be in header, since it's only need by the c file

Reply:
It has been modified in the new patch.